### PR TITLE
fix EdgeConfig version compare logic

### PR DESCRIPTION
**Pull request by jagadishmurugan**
fix EdgeConfig version compare logic

### CODE REVIEW

**Comment by jagadishmurugan** [comment]
one more change coming in to bump up config schema versions. will bump up QuickStart script version along with that change...

File: tools/modules/AksEdgeDeploy/AksEdgeDeploy.psm1 [hunk 1]
```diff
@@ -280,7 +280,10 @@ function UpgradeJsonFormat {
     #upgrade from public preview format to GA format
     $edgeCfg = $jsonObj.AksEdgeConfig
 
-    if ($edgeCfg.SchemaVersion -gt "1.4") {
+    $version = [int[]]$edgeCfg.SchemaVersion.Split(".")
+    $majorVersion = [int]$version[0]
+    $minorVersion = [int]$version[1]
+    if (($majorVersion -gt 1) -or ($majorVersion -eq 1 -and $minorVersion -gt 4)) {
```
Comment (seanyen) [inline tools/modules/AksEdgeDeploy/AksEdgeDeploy.psm1:286]: nit: you may want to use System.Version for generic version comparison logic: https://learn.microsoft.com/en-us/dotnet/fundamentals/runtime-libraries/system-version

**Review by seanyen — APPROVED** [review]
LGTM.

File: tools/modules/AksEdgeDeploy/AksEdgeDeploy.psm1 (hunk 1 above)
Comment (jagadishmurugan) [reply tools/modules/AksEdgeDeploy/AksEdgeDeploy.psm1]: used System.Version

**Pull request merged**